### ISSUE TITLE
Does Coordconv play a key role when training?

### ISSUE THREAD

**Issue by nebuladream** [Arthur151/CenterHMR#18]
I removed Coordconv when training centermap task independently, the training loss similar to the network with Coordconv, but the centermap seems to get bad on the test set.

**Comment by Arthur151** [comment]
I haven't made the ablation study of coordconv. But it makes sense that coordconv would be helpful in this cases.

**Comment by nebuladream** [comment]
Thank you, I will check this problem with more experiments.
